operations/app/src/modules/database/main.tf

```diff
@@ -23,7 +23,40 @@ resource "azurerm_postgresql_server" "postgres_server" {
   version = "11"
   storage_mb = 5120
 
-  auto_grow_enabled = (var.environment == "prod" ? true : false)
+  auto_grow_enabled = true
```

Comment:
Why the change to allow auto_grow in all environments?

Reply:
Auto-grow is a required condition for the read replica so that replication does not stop due to inadequate storage space.  Azure recommends keeping settings in sync between the master and replica, so I enabled it across the board.